clients/python-wrapper/tests/system_tests/test_sanity.py

```diff
@@ -1,16 +1,38 @@
-import os
+import http
+import time
+
 from lakefs import client
+from lakefs.exceptions import ServerException, RepositoryNotFoundException
+from lakefs.repository import Repository
+from tests.system_tests.conftest import expect_exception_context
 
 
-def test_sanity_client():
+def test_repository_sanity(storage_namespace):
     clt = client.DefaultClient
-    assert clt is not None
-    key = os.getenv("LAKECTL_CREDENTIALS_ACCESS_KEY_ID")
-    secret = os.getenv("LAKECTL_CREDENTIALS_SECRET_ACCESS_KEY")
-    host = os.getenv("LAKECTL_SERVER_ENDPOINT_URL")
-    assert clt.config.username == key
-    assert clt.config.password == secret
-    assert clt.config.host == host + "/api/v1"
-
-    version = os.getenv("TAG")
-    assert clt.version_config.version == version
+    repo_name = "my-repo"
+    default_branch = "main"
+    repo = Repository(repo_name, clt)
+    repo_data = repo.create(storage_namespace, default_branch, True)
+    now = time.time()
+    assert repo_data.id == repo_name
+    assert repo_data.storage_namespace == storage_namespace
+    assert float(repo_data.creation_date) <= now
```

Comment:
both work with seconds? do we need to use float?

Reply:
For seconds int is good enough,
```
    time() -> floating point number
    
    Return the current time in seconds since the Epoch.
    Fractions of a second may be present if the system clock provides them.
   ```